### PR TITLE
copied all 2.2.0 prerelase changelogs to 2.2.0 changelog 

### PR DESCRIPTION
**Pull request by VysotskiVadim**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,11 +8,51 @@ Mapbox welcomes participation and contributions from everyone.
 
 For details on how v2 differs from v1 and guidance on migrating from v1 of the Mapbox Navigation SDK for Android to the v2 public preview, see [2.0 Navigation SDK Migration Guide](https://github.com/mapbox/mapbox-navigation-android/wiki/2.0-Navigation-SDK-Migration-Guide).
 
+[Changes between v2.1.0 and v2.2.0](https://github.com/mapbox/mapbox-navigation-android/compare/v2.1.0...v2.2.0):
+
 ### Changelog
+#### Features
```
Comment (LukasPaczos) [inline CHANGELOG.md:14]:
```suggestion
### Changelog
[Changes between v2.1.0 and v2.2.0](https://github.com/mapbox/mapbox-navigation-android/compare/v2.1.0...v2.2.0):

#### Features
```

Wouldn't this be clearer?

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,11 +8,51 @@ Mapbox welcomes participation and contributions from everyone.
 
 For details on how v2 differs from v1 and guidance on migrating from v1 of the Mapbox Navigation SDK for Android to the v2 public preview, see [2.0 Navigation SDK Migration Guide](https://github.com/mapbox/mapbox-navigation-android/wiki/2.0-Navigation-SDK-Migration-Guide).
 
+[Changes between v2.1.0 and v2.2.0](https://github.com/mapbox/mapbox-navigation-android/compare/v2.1.0...v2.2.0):
```
Comment (LukasPaczos) [inline CHANGELOG.md:11]:
```suggestion
[Changes between v2.1.2 and v2.2.0](https://github.com/mapbox/mapbox-navigation-android/compare/v2.1.2...v2.2.0):
```

Comment (VysotskiVadim) [reply CHANGELOG.md]: Fixed, thanks

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,11 +8,51 @@ Mapbox welcomes participation and contributions from everyone.
...
+- Refactored extension `RouteOptions.Builder#applyDefaultNavigationOptions`, might be set profile param explicitly. [#5322](https://github.com/mapbox/mapbox-navigation-android/pull/5322)
+- Exposed a new API `MapboxManeuverView.updateLaneGuidanceIconStyle` that would allow changing the style of `MapboxLaneGuidanceAdapter` at runtime. [#5334](https://github.com/mapbox/mapbox-navigation-android/pull/5334)
+- Fixed a crash when `MapboxNavigationViewportDataSourceDebugger.enabled` is repeatedly set to true. [#5347](https://github.com/mapbox/mapbox-navigation-android/pull/5347)
+- Implemented vanishing route line feature from 1.x for exposing an option to adjust/limit the frequency of the vanishing route line updates. The MapboxRouteLineOptions.vanishingRouteLineUpdateIntervalNano can reduce the frequency of vanishing route line updates when the value of the option increases. [#5344](https://github.com/mapbox/mapbox-navigation-android/pull/5344)
+- Fixed `RoadShield` by reverting the breaking changes and use the new shield callback. [#5302](https://github.com/mapbox/mapbox-navigation-android/pull/5302)
```
Comment (LukasPaczos) [inline CHANGELOG.md:34]: This can be removed, since the braking changes were introduced and reverted during the course of the pre-releases.

**Review by LukasPaczos — APPROVED** [review]
LGTM with the one comment above to remove unnecessary entry.

**Pull request merged**